sql/telemetry/telemetry_core_parquet_v3/view.sql

```diff
@@ -1,12 +1,45 @@
--- Query generated by: templates/unnest_parquet_view.sql.py telemetry_core_parquet_v3 telemetry_derived.telemetry_core_parquet_v3
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.telemetry_core_parquet_v3` AS
 SELECT
-  submission_date AS submission_date_s3,
-  * REPLACE (
-    ARRAY(SELECT * FROM UNNEST(accessibility_services.list)) AS accessibility_services,
-    ARRAY(SELECT * FROM UNNEST(experiments.list)) AS experiments,
-    searches.key_value AS searches
-  )
+  DATE(submission_timestamp) AS submission_date_s3,
+  DATE(submission_timestamp) AS submission_date,
+  metadata.uri.app_name,
+  os,
+  STRUCT(
+    document_id,
+    UNIX_MICROS(submission_timestamp) * 1000 AS `timestamp`,
+    metadata.header.date AS `date`,
```

Comment:
this suggests to me that we have the date header, but for the backfill query it's null:
```CAST(NULL AS STRUCT<date STRING, dnt STRING, x_debug_id STRING, x_pingsender_version STRING>) AS header```

Reply:
You are very right. I will have to repeat the backfill and fill that in. Should just take a few hours.

Reply:
Backfill done. Merging.